modules/service-catalog/stack.py

```diff
@@ -0,0 +1,91 @@
+import os
+from typing import Any, cast
+
+import aws_cdk.aws_iam as iam
+import aws_cdk.aws_s3_assets as assets
+import aws_cdk.aws_servicecatalog as servicecatalog
+import cdk_nag
+from aws_cdk import Aspects, Stack, Tags
+from constructs import Construct, IConstruct
+
+from service_catalog.products import products
+
+
+class ServiceCatalogStack(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        *,
+        deployment_name: str,
+        module_name: str,
+        portfolio_access_role_arn: str,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(
+            scope,
+            id,
+            **kwargs,
+        )
+
+        Tags.of(scope=cast(IConstruct, self)).add(
+            key="Deployment",
```

Comment:
please refer to the deployment and module name in this tag, follow similar to the other addf modules